network/iptables_test.go

```diff
@@ -133,8 +133,8 @@ func TestDeleteRules(t *testing.T) {
 	if err != nil {
 		t.Error("Error setting up iptables")
 	}
-	if len(ipt.rules) != 6 {
-		t.Errorf("Should be 6 masqRules, there are actually %d: %#v", len(ipt.rules), ipt.rules)
+	if len(ipt.rules) != 7 {
+		t.Errorf("Should be 7 masqRules, there are actually %d: %#v", len(ipt.rules), ipt.rules)
```

Comment:
why 7? The test is done without the multiClusterCIDR enabled the rules should be the same.

Reply:
in fact I added a rule even when multiClusterCIDR is not used.
Instead of 1 rule that matches `source: cluster_cidr and dest: cluster_cidr`, we end up with 2 rules:
- `source: cluster_cidr and dest: pod_cidr`
- `source: pod_cidr and dest: cluster_cidr`
In practice, it is the same when there is only 1 cluster CIDR but it is more coherent with multiple cluster CIDRs.